packages/schematics/angular/library/index.ts

```diff
@@ -61,42 +53,35 @@ function updateJsonFile<T>(host: Tree, path: string, callback: UpdateJsonFn<T>):
   return host;
 }
 
-function updateTsConfig(npmPackageName: string, entryFilePath: string) {
-
-  return (host: Tree) => {
-    if (!host.exists('tsconfig.json')) { return host; }
-
-    return updateJsonFile(host, 'tsconfig.json', (tsconfig: TsConfigPartialType) => {
-      tsconfig.compilerOptions.baseUrl = '.';
-      if (!tsconfig.compilerOptions.paths) {
-        tsconfig.compilerOptions.paths = {};
-      }
-      if (!tsconfig.compilerOptions.paths[npmPackageName]) {
-        tsconfig.compilerOptions.paths[npmPackageName] = [];
-      }
-      tsconfig.compilerOptions.paths[npmPackageName].push(entryFilePath);
-    });
-  };
-}
-
 function addDependenciesAndScriptsToPackageJson() {
 
   return (host: Tree) => {
     if (!host.exists('package.json')) { return host; }
 
     return updateJsonFile(host, 'package.json', (json: PackageJsonPartialType) => {
+      if (!json['dependencies']) {
+        json['dependencies'] = {};
+      }
+
+      json.dependencies = {
+        '@angular/common': '^5.2.0',
+        '@angular/core': '^5.2.0',
+        '@angular/compiler': '^5.0.0',
```

Comment:
These versions should be consistent and ideally not repeated to simplify updates moving forward.

Reply:
Will fix, this and the TS one below.